packages/experimental/Button/src/Button.styling.ts

```diff
@@ -24,6 +23,7 @@ export const buttonStates: (keyof ButtonTokens)[] = [
   'rounded',
   'circular',
   'square',
+  'focused',
```

Comment:
I think you can get this + hovered + pressed by doing a spread of Pressable.states or something similar. But this is a string so I'm not sure

Reply:
That's true, although the ordering here actually matters so I'm not sure if adding a spread of all those states at the end is the best idea. I can do something like `...IFocusState.keys()` I think, so that we make sure we stay in sync with whatever that state is. I'll make that change separately, since other components would probably benefit from that change, too.